dallinger/command_line/docker_ssh.py

```diff
@@ -208,7 +208,7 @@ def wrapper(*args, **kwargs):  # pragma: no cover
             try:
                 check_output(["docker", "manifest", "inspect", image_name])
                 print(f"Image {image_name} found on remote registry")
-                return f(*args, **kwargs)
+                return f(*args, **dict(kwargs, image_name=image_name))
```

Comment:
Do you also need to update line 222 in the same way? I struggled to understand how this code worked before this change - how was the `deploy` function getting hold of `image_name`?